installing/installing_gcp/preparing-to-install-on-gcp.adoc

```diff
@@ -0,0 +1,48 @@
+
+* **xref:../../installing/installing_gcp/installing-gcp-network-customizations.adoc#installing-gcp-network-customizations[Installing a cluster on GCP with network customizations]**: You can customize your {product-title} network configuration during installation, so that your cluster can coexist with your existing IP address allocations and adhere to your network requirements.
+
+* **xref:../../installing/installing_gcp/installing-restricted-networks-gcp.adoc#installing-restricted-networks-gcp[Installing a cluster on GCP by using an internal mirror]**: You can install {product-title} on GCP infrastructure that you provide by using an internal mirror of the installation release content. You can use this method to install a cluster that does not require an active Internet connection to obtain the software components. While you can install {product-title} by using the mirrored content, your cluster still requires Internet access to use the GCP APIs.
+
+* **xref:../../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[Installing a cluster on an existing Virtual Private Cloud]**: You can install {product-title} on an existing GCP Virtual Private Cloud (VPC). You can use this installation method if you have constraints set by the guidelines of your company, such as limits when creating new accounts or infrastructure.
+
+* **xref:../../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp-private[Installing a private cluster on an existing VPC]**: You can install a private cluster on an existing GCP VPC. You can use this method to deploy {product-title} on an internal network that is not visible to the Internet.
+
+* **xref:../../installing/installing_gcp/installing-gcp-user-infra.adoc#installing-gcp-user-infra[Installing a cluster on GCP infrastructure that you provide]**: You can install {product-title} on GCP infrastructure that you provide. You can use the provided Deployment Manager templates to assist with the installation.
+
+* **xref:../../installing/installing_gcp/installing-restricted-networks-gcp.adoc#installing-restricted-networks-gcp[Installing a cluster on GCP in a restricted network with user-provisioned infrastructure]**: You can install {product-title} on GCP in a restricted network with user-provisioned infrastructure. By creating an internal mirror of the installation release content, you can install a cluster that does not require an active Internet connection to obtain the software components. You can also use this installation method to ensure that your clusters only use container images that satisfy your organizational controls on external content.
+
+* **xref:../../installing/installing_gcp/installing-gcp-user-infra-vpc.adoc#installing-gcp-user-infra-vpc[Installing a cluster with shared VPC on user-provisioned infrastructure in GCP by using Deployment Manager templates]**: You can use the provided templates to create GCP resources in a shared VPC infrastructure.
```

Comment:
Do we need _by using Deployment Manager templates_ in the bullet heading? Seems to suggest there is another way to install a cluster with shared VPC on user-provisioned infrastructure in GCP. Maybe move the phrase into the text? 
_You can use the provided Deployment Manager templates to..._

Reply:
I completely, 1000% agree with your thoughts on this. I left it that way  because that's the title and I wasn't sure I should change it - my mission is simply the "Preparing to..." page. I'm going to make your suggested change, then ponder a possible change to the title it links to....